cli/internal/run/real_run.go

```diff
@@ -123,6 +123,11 @@ func RealRun(
 	// Assign tasks after execution
 	runSummary.Tasks = taskSummaries
 
+	// We walk the graph with no concurrency.
+	// Populating the cache state is parallelizable.
+	// Do this _after_ walking the graph.
+	runSummary.PopulateCacheState(turboCache)
```

Comment:
It would be nice if we could do this, but we can't because it's after executing the task, so the cache is already populated and should be true. We want values from the ec.exec`